tools/gfx-unit-test/get-queue-handle-test.cpp

```diff
@@ -0,0 +1,81 @@
+#include "tools/unit-test/slang-unit-test.h"
+
+#include "slang-gfx.h"
+#include "gfx-test-util.h"
+#include "tools/gfx-util/shader-cursor.h"
+#include "source/core/slang-basic.h"
+
+#if SLANG_WINDOWS_FAMILY
+#include <d3d12.h>
+#endif
+
+using namespace gfx;
+
+namespace gfx_test
+{
+    void getQueueHandleTestImpl(IDevice* device, UnitTestContext* context)
+    {
+        ICommandQueue::Desc queueDesc = { ICommandQueue::QueueType::Graphics };
+        auto queue = device->createCommandQueue(queueDesc);
+        ICommandQueue::NativeHandle handle;
+        GFX_CHECK_CALL_ABORT(queue->getNativeHandle(&handle));
+        if (device->getDeviceInfo().deviceType == gfx::DeviceType::Vulkan)
+        {
+            SLANG_CHECK(handle != 0);
```

Comment:
Depending what the native handle is perhaps 0 is a valid value. If 0 is defined to be an invalid handle it should perhaps be documented as such.

Reply:
`VK_NULL_HANDLE` is defined to be 0. I'll add a comment.